app/scripts/modules/core/pipeline/config/stages/executionWindows/ExecutionWindowActions.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from 'react';
+
+import {IExecution} from 'core/domain/IExecution';
+import {IExecutionStage} from 'core/domain/IExecutionStage';
```

Comment:
I've started combining all imports from `core/domain/*` into a single import like:
`import {IExecution, IExecutionStage} from 'core/domain';

If we have the index.ts file... let's use it.